csharp_dotnetcore/8.Suggested-Actions/BotBuilder.ruleset

```diff
@@ -0,0 +1,22 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I see this file twice. Once in "8.Suggested-Actions" and once in "8.Suggested_Actions". 

Please remove the incorrect file .